tests/test_models.py

```diff
@@ -99,6 +100,40 @@ def test_load():
         TileMatrixSet.load("ANotValidName")
 
 
+def test_quadkey_support():
+    tms = TileMatrixSet.load("CanadianNAD83_LCC")
+    assert not tms.quadkey_support
+
+    tms = TileMatrixSet.load("UPSArcticWGS84Quad")
+    assert tms.quadkey_support
+
```

Comment:
Please add test cases for calling both quadkey methods on a tile matrix set that doesn't support quadkeys.  We want to test that behavior to prevent the code from regressing.

The grid mentioned by @blacha in the original issue would probably be a good test case.